playbooks/tasks/logging/configure_logging_agent.yml

```diff
@@ -0,0 +1,181 @@
+---
+- name: Configure Logging Agent
+  hosts: hetzner
```

Comment:
We don't use a `hetzner` group. Host should be either `all` or `beacon, execution`